austation/code/modules/hydroponics/grown.dm

```diff
@@ -32,3 +44,13 @@
 		reagents.reaction(A)
 
 	qdel(src)
+
+/obj/item/reagent_containers/food/snacks/grown/unembedded()
+	. = ..()
+	for(var/datum/plant_gene/trait/spines/S in seed.genes)
```

Comment:
can there be more than one??
applies to the other for loops as well
surely you can just use get_gene() like you do everywhere else